### ISSUE TITLE
taskforce-connector hangs

### ISSUE THREAD

**Issue by sarendipitee** [taskforcesh/issues#6]
Doesn't work or do anything, simply hangs. Nothing shows up in taskforce web ui.

```
$ taskforce -n 'dev' 0e68fe3e-<redacted>-9e2c975fb888
Taskforce Connector v1.0.2 - (c) 2017-2018 Taskforce.sh Inc.
```

Redis running on localhost

```
ID                  NAME                  MODE                REPLICAS            IMAGE                                  PORTS
iklhtbj9a6vg        api_redis             replicated          1/1                 redis:5.0.2-alpine                     *:6379->6379/tcp
```

Other apps connect fine @ localhost:6379

**Comment by manast** [comment]
In the example above you are missing the ```-t``` option, not sure if that is the cause for the issue, you should have got an error in any case.
Example:
```
taskforce -n "transcoder connection" -t 2cfe6a1b-5f0e-466f-99ad-12f51bea79a7
```

**Comment by sarendipitee** [comment]
That was it, thanks. Definitely no error when that arg was missing though.

**Comment by manast** [comment]
Fixed in taskforce-connector release v1.0.3
